packages/tools/client-debug-view/src/components/AudienceView.tsx

```diff
@@ -27,6 +28,10 @@ export interface AudienceViewProps {
      */
     audience: IServiceAudience<IMember>;
 
+    /**
+     * Audience
+     */
+    history: AudienceHistory[];
```

Comment:
Nit: can we add a blank line between this and the next property?

Reply:
Added.